tests/test_datasetattributes.py

```diff
@@ -51,35 +52,126 @@ def test_init(hexbeam):
     assert attributes.association == FieldAssociation.POINT
 
 
+def test_repr(hexbeam_point_attributes):
+    repr_str = str(hexbeam_point_attributes)
+    assert 'POINT' in repr_str
+    assert 'DataSetAttributes' in repr_str
+    assert 'Contains arrays' in repr_str
+    assert '...' not in repr_str
+
+    # ensure long names are abbreviated
+    first_arr = hexbeam_point_attributes.keys()[0]
+    sz = hexbeam_point_attributes[first_arr].size
```

Comment:
```suggestion
    sz = hexbeam_point_attributes.values()[0].size
```

Reply:
`first_arr` was used in the old API.  Thanks!